File: StraviaTec.API/Models/Actividad.cs
using System.ComponentModel.DataAnnotations;
using System.Xml;

namespace StraviaTec.API.Models
{
    public class Actividad
    {
        [Key]
        public int id { get; set; }
        public int? id_reto { get; set; }
        public string id_usuario { get; set; }
        public TimeSpan hora_inicio { get; set; }
        public TimeSpan hora_fin { get; set; }
        public string tipo_actividad { get; set; }
        public int kilometraje { get; set; }
        public DateTime fecha { get; set; }
        public string? recorrido { get; set; }
    }
}

File: AppWeb/src/app/Models/actividad.ts
import { Time } from "@angular/common"

export class Actividad {
    id:number | undefined
    id_reto:number | undefined
    id_usuario:string | undefined
    hora_inicio:Time | undefined
    hora_fin:Time | undefined
    tipo_actividad:string | undefined
    kilometraje:number | undefined
    fecha:Date | undefined
    recorrido:string | undefined
}
